forc-gm/Cargo.toml

```
@@ -0,0 +1,12 @@
+[package]
+name = "forc-gm"
+version = "0.11.3"
```

```suggestion
version = "0.11.0"
```
^ This should match `forc`'s version

---

Oops! My bad